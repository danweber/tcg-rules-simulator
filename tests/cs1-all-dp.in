VERSION:2
# CS1-Weakpoint:
# [Main] All of your opponent's Monsters get -5000 DP for the turn.

FIELD1:YellowGreen,CS1-08
FIELD2:Two,CS2-03 Three,CS2-03 Four,CS2-04 Five,CS2-05
HAND1:CS1-WeakPoint
# LongSword will get played from security
SECURITY2:CS2-LongSword
GAME:P1 T3 MAIN 8
DUMP:SECURITY,DP

ECHO: -- Use Weakpoint
CMD:PLAY CS1-Weakpoint
STEP:AUTOANSWER:CAPTURE
DUMP:SECURITY,DP


ECHO: -- P1 Attacks to bring out LongSword, it should die. (Also Phoenix Shield resurrects)
CMD:ATTACK YellowGreen Player
STEP:AUTOANSWER:CAPTURE
DUMP:SECURITY,DP



