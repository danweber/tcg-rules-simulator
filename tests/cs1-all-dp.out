FIELD1:YellowGreen,CS1-Phoenixdefender,1000DP
FIELD2:Two,CS2-Longsword,1500DP Three,CS2-Longsword,1500DP Four,CS2-Rabiddog,4500DP Five,CS2-Greatsword,5500DP
SECURITY1:
SECURITY2:CS2-Longsword
GAME:P1 T3 MAIN 8
ECHO: -- Use Weakpoint
FIELD1:YellowGreen,CS1-Phoenixdefender,1000DP
FIELD2:Five,CS2-Greatsword,500DP
SECURITY1:
SECURITY2:CS2-Longsword
GAME:P1 T3 MAIN 1
ECHO: -- P1 Attacks to bring out LongSword, it should die. (Also Phoenix Shield resurrects)
FIELD1:I7,CS1-Phoenixdefender,1000DP
FIELD2:Five,CS2-Greatsword,500DP
SECURITY1:
SECURITY2:
GAME:P1 T3 MAIN 1
